### DISCUSSION TITLE
My Mapbox Theme

### DISCUSSION THREAD

**Discussion by u-an-i** [albar965/littlenavmap#1149] [Show and tell]
**Preview:**
https://api.mapbox.com/styles/v1/uai123/cllk700tc00vu01qp48al0mek.html?title=view&access_token=pk.eyJ1IjoidWFpMTIzIiwiYSI6ImNsbzFwd256OTAyYWoyaW14aDZxYWRweGYifQ.gBrCrPHrCN8ekqfLjEU_Qw&zoomwheel=true&fresh=true#15.45/48.998156/8.4602

**Copy for your Mapbox account:**
https://api.mapbox.com/styles/v1/uai123/cllk700tc00vu01qp48al0mek.html?title=copy&access_token=pk.eyJ1IjoidWFpMTIzIiwiYSI6ImNsbzFwd256OTAyYWoyaW14aDZxYWRweGYifQ.gBrCrPHrCN8ekqfLjEU_Qw&zoomwheel=true&fresh=true#15.45/48.998156/8.4602

**My Style url:**
mapbox://styles/uai123/cllk700tc00vu01qp48al0mek

**Demo:**
https://github.com/albar965/littlenavmap/assets/84718885/6d9f857f-190d-4eba-8467-9368ca835c5c

**Picture:**
![Screenshot 2024-06-08 101525](https://github.com/albar965/littlenavmap/assets/84718885/82159679-2fc3-491a-9ca4-f6cfb3f0c899)

**Comment by albar965** [comment]
Quite colorful:
![image](https://github.com/albar965/littlenavmap/assets/15254956/628f4925-973f-4ae2-9750-a906f455f1de)

I have one too:
[Flying Terrain Map](https://api.mapbox.com/styles/v1/albar965/ckyre6plz16wr14lhncty6zzm.html?title=copy&access_token=pk.eyJ1IjoiYWxiYXI5NjUiLCJhIjoiY2t5cmU1eG9nMHNtYzJ2bXYxNzlrYzl1NCJ9.W-PUTe_AMBr_6BFcuiED6g&zoomwheel=true&fresh=true#12.16/46.05216/9.12467)

**Reply by u-an-i** [reply]
👀 😄
